Issue 78 docs

mikekeke opened the pull request:
Better documentation.
README reworked, new tutorials added.

zmrocze commented:
Back when I was working out how to create the `cluster-data` config files for `vasil-local-cluster` I wrote a small doc (`cluster-data/regenerate-configs.md`)
So maybe it makes sense to keep it? and now is a good moment to add it. But feel free to revert last commit where I added it.

A detail: that doc mentioned 'print-byron-mainnet-addresses.zsh' helper script that is also included in the `vasil-local-cluster` branches. But I thought its too unpolished and removed the mention, little likely that anyone would use it (?).

zmrocze commented:
Good docs!

mikekeke commented:
@zmrocze 
Great addition IMO.
I moved readme to `docs` to keep `cluster-data` closer to content of corresponding `cardano-wallet` dir and added separate section for network configuring in main readme.

mikekeke commented on docs/tasty-integration.md:
```diff
@@ -0,0 +1,200 @@
+* `initAdaAssertValue [100] 133` - initialize `wallet` with single UTxO with 100 Ada and check that after contract execution final `Value` of all `wallet`'s UTxOs is equal to 133 Ada.
+* `initAndAssertLovelaceWith [1_000_000] VGt 2_000_000` - initialize `wallet` with single UTxO with 1000000 Lovelace and check that after contract execution final `Value` of all `wallet`'s UTxOs is *greater than* 2000000 Lovelace.
+
+***One important note*** is that Plutip handles creation of collateral UTxO under the hood. So when using assertions for `Value` it is advised to wrap `wallet`'s initialization with `withCollateral` function, like this:
```
Fixed in #[7b86507](https://github.com/mlabs-haskell/plutip/pull/133/commits/7b865072ebaddc31f93127359efbe505819fe65b)
